src/lib/tagHelper.ts

```diff
@@ -0,0 +1,20 @@
+const hasEventBubbledThroughTag = ({
```

Comment:
if you only use this function in one place, I'd not introduce a helper for it just yet, but bundle the function with the component